webapp/src/DataFixtures/DefaultData/ExecutableFixture.php

```diff
@@ -0,0 +1,89 @@
+<?php declare(strict_types=1);
+
+namespace App\DataFixtures\DefaultData;
+
+use App\Entity\Executable;
+use App\Entity\ImmutableExecutable;
+use App\Service\DOMJudgeService;
+use Doctrine\Persistence\ObjectManager;
+use ZipArchive;
+
+class ExecutableFixture extends AbstractDefaultDataFixture
+{
+    /**
+     * @var string
+     */
+    protected $sqlDir;
+
+    /**
+     * @var DOMJudgeService
+     */
+    protected $dj;
+
+    public function __construct(string $sqlDir, DOMJudgeService $dj)
+    {
+        $this->sqlDir = $sqlDir;
+        $this->dj     = $dj;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function load(ObjectManager $manager)
+    {
+        $data = [
+            // ID,                description,                                         type
+            ['adb',               'adb',                                              'compile'],
+            ['awk',               'awk',                                              'compile'],
+            ['bash',              'bash',                                             'compile'],
+            ['c',                 'c',                                                'compile'],
+            ['compare',           'default compare script',                           'compare'],
+            ['cpp',               'cpp',                                              'compile'],
+            ['csharp',            'csharp',                                           'compile'],
+            ['f95',               'f95',                                              'compile'],
+            ['float',             'default compare script for floats with prec 1E-7', 'compare'],
+            ['hs',                'hs',                                               'compile'],
+            ['kt',                'kt',                                               'compile'],
+            ['java_javac',        'java_javac',                                       'compile'],
+            ['java_javac_detect', 'java_javac_detect',                                'compile'],
+            ['js',                'js',                                               'compile'],
+            ['lua',               'lua',                                              'compile'],
+            ['pas',               'pas',                                              'compile'],
+            ['pl',                'pl',                                               'compile'],
+            ['plg',               'plg',                                              'compile'],
+            ['py2',               'py2',                                              'compile'],
```

Comment:
Time to drop this?

Reply:
I am fine with this. What do others think?

Reply:
Not blocking for this PR.

Reply:
Removed it.